clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreBufferService.java

```diff
@@ -532,7 +532,7 @@ public long getMaxMemoryUsagePerDrainer() {
         LOGGER.info(
             "Drainer number {} hosting {} partitions, has memory usage of {}",
             index,
-            drainer.topicToTimeSpent.size(),
+            drainer.blockingQueue.size(),
```

Comment:
The `drainer.blockingQueue.size()` is the number of messages queued up, from my understanding, not the number of partitions... Do you believe that `drainer.topicToTimeSpent.size()` is incorrect?

Reply:
Getting rid of the log SGTM. I think we already have a metric for that anyway.